### ISSUE TITLE
Release 3.0

### ISSUE THREAD

**Issue by dconnolly** [ZcashFoundation/ed25519-zebra#45]
Upgrading to rand 0.8 / rand_core 0.6 (#40) makes our public api incompatible with 2.0.

**Comment by kim** [comment]
How do you feel about pushing out a minor release with the changes before the `rand` upgrade?

**Comment by teor2345** [comment]
> How do you feel about pushing out a minor release with the changes before the `rand` upgrade?

The Zeroize (#34) and JNI (#37) changes?

Happy to do that if it would be helpful. Releases don't take a lot of effort - we just need to make time for them.

**Comment by kim** [comment]
Yeah would be appreciated. I realised we're still on the git dependency (for #34), and I'm not sure if the `rand` upgrade would have effects on other dependencies. Thanks!

**Comment by FintanH** [comment]
Hey @teor2345 :wave: I wanted to ping about the above minor release. We're still pinning on #34 and it'd be nice to pin to a version. Anything I could do to help? :blush:

**Comment by dconnolly** [comment]
@FintanH I'm scheduling a release within the next week or so. 📦

**Comment by FintanH** [comment]
Amazing, thanks @dconnolly! :bow:

**Comment by FintanH** [comment]
@dconnolly: did you ever get around to that release? :eyes: Just remembered this after figuring out why some types were different across builds :sweat_smile:
